Sources/KlaviyoSwift/KlaviyoInternal.swift

```diff
@@ -151,6 +153,28 @@ package enum KlaviyoInternal {
         profileDataSubject.send(.failure(.notInitialized))
     }
 
+    // MARK: - Profile Event methods
```

Comment:
every time I see these MARK comments my brain thinks I'm about to read a note from our PM 😂